jetty9-compat-base/src/test/java/com/google/apphosting/runtime/jetty9/SessionManagerIT.java

```diff
@@ -68,12 +62,18 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpSession;
 
+import static org.easymock.EasyMock.createMock;
```

Comment:
Not sure why this was changed. static imports should be at the top according to the [Google style guide](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing)